### PR TITLE
Removes Jaunt invocation, speech bubbles don't follow invisible casters

### PR DESCRIPTION
**Pull request by 9600bauds**
Jaunt was having problems in that the invocation caused a speech bubble, which followed the invisible wizard while they jaunted
I thought that by making invisible mobs have speech bubbles not follow them, the problem would be solved
Sadly the actual jaunt invocation happens before the jaunt, so I just made jaunt not have an invocation again
Left the whole "speech bubbles don't follow invisible casters" thing in in case a brain damage wizard speaks while jaunted or something
:cl: 
 * bugfix: Removed the verbal invocation from Ethereal Jaunt, because it was following the invisible wizard and giving away where they jaunted to.

### CODE REVIEW

**Comment by Intigracy** [comment]
No way to set JUST the invisibility before the invocation without too much work?

**Comment by Intigracy** [comment]
Actually there is.

Here's the corresponding spellcode:

```
/spell/proc/perform(mob/user = usr, skipcharge = 0, list/target_override) //if recharge is started is important for the trigger spells
	if(!holder)
		holder = user //just in case

	var/list/targets = target_override

	if(before_channel(user))
		return
	if(!targets && (spell_flags & WAIT_FOR_CLICK))
		channel_spell(user, skipcharge)
		return
	if(!cast_check(skipcharge, user))
		return
	if(cast_delay && !spell_do_after(user, cast_delay))
		return
	if(before_target(user))
		return

	if(!targets)
		targets = choose_targets(user)

	if(!cast_check(skipcharge, user))
		return //Prevent queueing of spells by opening several choose target windows.
	if(targets && targets.len)
		targets = before_cast(targets, user) //applies any overlays and effects<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
		if(!targets.len) //before cast has rechecked what we can target
			return
		invocation(user, targets) <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<

		user.attack_log += text("\[[time_stamp()]\] <font color='red'>[user.real_name] ([user.ckey]) cast the spell [name].</font>")
		INVOKE_EVENT(user.on_spellcast, list("spell" = src, "target" = targets))

		if(prob(critfailchance))
			critfail(targets, user)
		else
			. = cast(targets, user) //return 1 to prevent take_charge
		if(!.)
			take_charge(user, skipcharge)
		after_cast(targets) //generates the sparks, smoke, target messages etc.
```

**Comment by Intigracy** [comment]
Simply modify before_cast to set the invisibility.

**Comment by 9600bauds** [comment]
Aauuuugh but the jaunt effect itself is like a whole page and the invisibility is only one line of it

**Comment by Shadowmech88** [comment]
If possible, I think it'd make most logical sense for the speech bubble to appear on a random tile near the invisible speaker. 
Just because a person's invisible doesn't mean you can't tell what direction their voice is coming from, after all.

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -414,16 +414,26 @@ var/list/department_radio_keys = list(
...
+				static_speech_bubble_recipients.Add(M.client)
+			else
+				tracking_speech_bubble_recipients.Add(M.client)
 	spawn(0)
-		var/image/speech_bubble = image('icons/mob/talk.dmi', get_holder_at_turf_level(src), "h[bubble_type][say_test(message)]",MOB_LAYER+1)
-		speech_bubble.plane = BASE_PLANE
-		speech_bubble.appearance_flags = RESET_COLOR
-		flick_overlay(speech_bubble, speech_bubble_recipients, 30)
+		if(static_speech_bubble_recipients.len)
+			var/image/speech_bubble = image('icons/mob/talk.dmi', get_turf(src), "h[bubble_type][say_test(message)]",MOB_LAYER+1)
```
Comment (ComicIronic) [inline code/modules/mob/living/say.dm]: Make a proc for the 4 lines, no copy paste.

Comment (9600bauds) [reply code/modules/mob/living/say.dm]: I can actually just save the location and use that here, I don't know what I was thinking

Comment (9600bauds) [reply code/modules/mob/living/say.dm]: Oh no wait I'm sending these to two different lists, I see what I was doing

**Comment by Intigracy** [comment]
Actually you could have just modified invocation() :(

**Comment by 9600bauds** [comment]
c my comment above

**Pull request merged**